src/main/scala/viper/gobra/Gobra.scala

```diff
@@ -119,24 +126,32 @@ trait GoVerifier extends StrictLogging {
 
     // Print statistics for caching
     if(config.cacheFile.isDefined) {
-      logger.debug(s"Number of cacheable Viper member(s): ${statsCollector.getNumberOfCacheableViperMembers}")
-      logger.debug(s"Number of cached Viper member(s): ${statsCollector.getNumberOfCachedViperMembers}")
+      logger.debug(s"Number of cacheable Viper members: ${statsCollector.getNumberOfCacheableViperMembers}")
+      logger.debug(s"Number of cached Viper members: ${statsCollector.getNumberOfCachedViperMembers}")
     }
 
     // Print general statistics
-    logger.debug(s"Gobra has found ${statsCollector.getNumberOfVerifiableMembers} methods and functions" )
-    logger.debug(s"${statsCollector.getNumberOfSpecifiedMembers} have specification")
-    logger.debug(s"${statsCollector.getNumberOfSpecifiedMembersWithAssumptions} are assumed to be satisfied")
+    logger.debug(s"Gobra found ${statsCollector.getNumberOfVerifiableMembers} methods and functions." )
+    logger.debug{
+      val specMem = statsCollector.getNumberOfSpecifiedMembers
+      s"$specMem member${addPlural(specMem)} are explicitly specified."
+    }
+    logger.debug{
+      val memAssum = statsCollector.getNumberOfSpecifiedMembersWithAssumptions
+      s"$memAssum member${addPlural(memAssum)} contain explicit assumptions."
+    }
```

Comment:
Out of curiosity, what are explicit members and explicit assumptions? Does this refer to members and assumptions in the Gobra program as opposed to in the encoding?